js/arithmetic-quiz.js

```diff
@@ -36,16 +39,23 @@ H5P.ArithmeticQuiz = (function ($) {
         incorrectText: 'Incorrect. Correct answer was :num',
         durationLabel: 'Duration in hours, minutes and seconds.',
         humanizedQuestion: 'What does :arithmetic equal?',
+        humanizedEquation: 'For the equation :equation, what does :item equal?',
+        humanizedVariable: 'What does :item equal?',
         plusOperator: 'plus',
         minusOperator: 'minus',
         multiplicationOperator: 'times',
         divisionOperator: 'divided by',
+        equalitySign: 'equal',
         slideOfTotal: 'Slide :num of :total'
       }
     }, options);
     self.currentWidth = 0;
 
-    self.gamePage = new H5P.ArithmeticQuiz.GamePage(self.options.arithmeticType, self.options.maxQuestions, self.options.UI, id);
+    // Reset equation type if not selected
+    if (self.options.quizType === H5P.ArithmeticQuiz.QuizType.ARITHMETIC) {
```

Comment:
I think this is unnecessary. Instead of checking arithmeticType/equationType, quizType should be used to indicate which mode we're in. That will make the code more readable.